### PR TITLE
zh, en: add missing task config

### PR DESCRIPTION
**Pull request by elvizlai**
### What is changed, added or deleted? (Required)

Add missing `case-sensitive` config for task.

### Which DM version(s) do your changes apply to? (Required)

- [ ] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: en/task-configuration-file-full.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ meta-schema: "dm_meta"          # The downstream database that stores the `meta`
 remove-meta: false              # Whether to remove the `meta` information (`checkpoint` and `onlineddl`) corresponding to the task name before starting the replication task.
 enable-heartbeat: false         # Whether to enable the heartbeat feature.
 online-ddl-scheme: "gh-ost"     # Only "gh-ost" and "pt" are currently supported.
+case-sensitive: false           # Whether to enable the case-sensitive feature for schema/table.
```
Comment (toutdesuite) [inline en/task-configuration-file-full.md:34]:
```suggestion
case-sensitive: false           # Whether schema/table is case-sensitive
```

Comment (toutdesuite) [inline en/task-configuration-file-full.md:34]:
```suggestion
case-sensitive: false           # Whether schema/table is case-sensitive.
```

Comment (toutdesuite) [inline en/task-configuration-file-full.md:34]:
```suggestion
case-sensitive: false           # Whether schema/table is case-sensitive.
```

**Comment by toutdesuite** [comment]
A minor suggestion. @elvizlai PTAL~

**Comment by toutdesuite** [comment]
@WangXiangUSTC Whether the changes apply to dev?

**Comment by WangXiangUSTC** [comment]
> @WangXiangUSTC Do the changes apply to dev?

The dev had this config.

**Pull request merged**